db/migrate/20150808115436_remove_requirement_from_users_invitation_code.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveRequirementFromUsersInvitationCode < ActiveRecord::Migration
+  def change
+    change_column_null :users, :invitation_code, true, ENV['INVITATION_CODE']
```

Comment:
Ah, ok!  Then do `ENV['INVITATION_CODE'].presence || 'try-huginn'` I think so that it uses our default.